static/src/javascripts-legacy/projects/facia/modules/ui/lazy-load-containers.js

```diff
@@ -16,30 +14,55 @@ define([
     var distanceBeforeLoad = detect.getViewport().height;
 
     return function () {
-        var $frontBottom = bonzo(qwery('.js-front-bottom')),
-            containers = qwery('.js-container--lazy-load'),
+        var containers = qwery('.js-container--lazy-load'),
             lazyLoad = throttle(function () {
                 if (containers.length === 0) {
                     mediator.off('window:throttledScroll', lazyLoad);
                 } else {
                     fastdom.read(function () {
-                        var scrollTop = window.pageYOffset,
-                            scrollBottom = scrollTop + bonzo.viewport().height,
-                            bottomOffset = $frontBottom.offset().top,
-                            $container;
-
-                        if (scrollBottom > bottomOffset - distanceBeforeLoad) {
-                            $container = bonzo(containers.shift());
-
-                            fastdom.write(function () {
-                                $container.removeClass('fc-container--lazy-load');
-                            });
-                        }
+                        var cs = split(containers, withinRange);
```

Comment:
`cs` isn't too clear, i can see you're crying out for destructuring! how about just `splitResult` for now?